etc/calc_release_version.py

```diff
@@ -22,6 +22,18 @@
 current Git commit.
 """
 
+# XXX NOTE XXX NOTE XXX NOTE XXX
+# After modifying this script it is advisable to execute the self-test.
+#
+# This is done by starting in the directory containing this script and then
+# executing a separate self-test script, like this:
+#
+# $ bash ../.evergreen/calc_release_version_selftest.sh
```

Comment:
Consider moving `calc_release_version_selftest.sh` to the `etc` directory so it is located next to `calc_release_version.py`. Since `calc_release_version_selftest.sh` is not run in Evergreen, I see no reason to keep it in the `.evergreen` directory.